tiledb/bioimg/converters/scale.py

```diff
@@ -12,261 +12,102 @@
 from .tiles import iter_tiles
 
 
-class ScaleMethod(Enum):
-    NEAREST = 0
-    LINEAR = 1
+class ScalerMode(IntFlag):
+    NON_PROGRESSIVE = auto()
+    PROGRESSIVE = auto()
+    CHUNKED_NON_PROGRESSIVE = auto()
+    CHUNKED_PROGRESSIVE = auto()
 
 
-class ScalerMode(Enum):
-    NON_PROGRESSIVE = 0
-    PROGRESSIVE = 1
-    CHUNKED_NON_PROGRESSIVE = 2
-    CHUNKED_PROGRESSIVE = 3
+def _scale_tile(
+    tile: Tuple[slice],
```

Comment:
This means a tuple of exactly one slice; the correct annotation is `Tuple[slice, ...]`